src/routes/+page.svelte

```diff
@@ -22,13 +23,18 @@
 
 <div>
     <h1 class="text-3xl font-bold underline">Hello world!</h1>
+    <TextField
+        textFieldText="Enter Recipient Address or EOS Account"
+        validator={(val) => /^0x[a-fA-F0-9]{40}$/.test(val) || /^[a-zA-Z0-9]+$/.test(val)}
+        errorText="Incorrect wallet format."
```

Comment:
nit: since this is now checking for account or wallet you may want to update the error text to say 
```incorrect format``` or ``` incorrect wallet/account format```

Reply:
Additionally, the criteria for accounts is "a-z,1-5 are allowed only. Length 12". Consider this:
```ts
const recipientAddressOrEosAccountRegex = /^(?:0x[a-fA-F0-9]{40})|(?:[a-z1-5]{1,12})$/;
// then later...
const validator = value => recipientAddressOrEosAccountRegex.test(value);
```